modules/book.js

```diff
@@ -0,0 +1,64 @@
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  // getbook  method
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    // console.log(books);
+    return books;
+  }
+
+  // addbook  method
+  static addBook(book) {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+    // console.log(books);
```

Comment:
Please remove unused commented code from the codebase it is not the best practice to leave commented code